go-controller/pkg/cni/ovs.go

```diff
@@ -307,12 +295,14 @@ func waitForPodInterface(ctx context.Context, mac string, ifAddrs []*net.IPNet,
 			if err := isIfaceIDSet(ifaceName, ifaceID); err != nil {
```

Comment:
@flavio-fernandes good idea, it could be a quicker check if a second sandbox has started after us. If we have already set iface-id-ver (which by this point we have) and then we re-read it and it has changed, that means kubelet/CRIO triggered a new sandbox request, and this one should exit.

Reply:
In CNI sandbox is associates with CNI_NETNS, and in the code PodRequest.SandboxID is set to CNI_CONTAINERID.
When you say "kubelet/CRIO triggered a new sandbox request" does it mean CNI request with the same K8S_POD_* arguments, but different CNI_CONTAINERID?

Reply:
If so, `iface-id-ver` will never change for given `ifaceName`, but `iface-id` will be cleaned by the next ADD request

Reply:
@npinaeva yes, that means same pod name + namespace but different NETNS. Our `iface-id-ver` should be the pod UUID, which *is* unique between requests.

Reply:
Yes, but what about CNI_CONTAINERID? because `ifaceName` is based on this variable

Reply:
@dcbw it's important for my next revert PR ^